lib/sync/storage-manager-listener.js

```diff
@@ -0,0 +1,36 @@
+let cachedStore;
+let cachedListener;
+
+module.exports = function getStorageManagerListener(store) {
+  if (cachedListener && (!store || store === cachedStore)) { return cachedListener; }
+
+  if (!store || (typeof store.dispatch !== 'function')) { throw new Error('You must pass in a redux store the first time you call getStorageManagerListener'); }
+
+  cachedStore = store;
```

Comment:
It's for specs - without this, we don't have a way of getting specs that can reset with a new store in every beforeEach within the same file